metricflow/test/integration/test_cases/itest_metrics.yaml

```diff
@@ -1127,3 +1127,182 @@ integration_test:
       , {{ render_extract("ds", DatePart.DOW) }} AS metric_time__extract_dow
     FROM {{ source_schema }}.fct_bookings
     GROUP BY {{ render_extract("ds", DatePart.DOW) }};
+---
+integration_test:
+  name: simple_fill_nulls_with_0_metric_time
+  description: Test a simple query that joins to time spine and fills nulls
+  model: SIMPLE_MODEL
+  metrics: ["bookings_fill_0"]
+  group_by_objs: [{"name": "metric_time"}]
+  check_query: |
+    SELECT
+      subq_5.ds AS metric_time__day
+      , COALESCE(subq_3.bookings, 0) AS bookings_fill_0
+    FROM {{ source_schema }}.mf_time_spine subq_5
+    LEFT OUTER JOIN (
+      SELECT
+        DATE_TRUNC('day', ds) AS metric_time__day
```

Comment:
I don't think this works for all engines, does it?

Reply:
Whoops missed that! Will update.